tendrl/commons/tests/objects/cluster/atoms/check_sync_done/test_check_sync_done_init.py

```diff
@@ -0,0 +1,30 @@
+import __builtin__
+import maps
+from tendrl.commons.objects.cluster.atoms.check_sync_done import CheckSyncDone
```

Comment:
Try keeping tendrl imports separate from library imports. We can have a line in between them.